rest-api-spec/src/main/resources/rest-api-spec/test/index/110_constant_keyword.yml

```diff
@@ -0,0 +1,70 @@
+---
+setup:
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            properties:
+              genre:
+                type : "constant_keyword"
+                "value" : "1"
```

Comment:
Small nit, `value` is not needed to quoted.

Reply:
What I want to test here is that value can be int or string.

Reply:
I mean the key: `"value"`.